### DISCUSSION TITLE
How to get status of each task?

### DISCUSSION THREAD

**Discussion by Nathanrown** [nextflow-io/nextflow#3921] [Q&A]
I only use `nextflow  log`` to get status of my pipeline, but how to get status of tasks in my pipeline ?

**Comment by mribeirodantas** [comment]
You can check the status of your tasks in your console output (even better with `-ansi-log false`), with the user interface of Nextflow Tower (if you run your Nextflow pipeline with Tower), or in the log files. If I run `nextflow run nextflow-io/hello`, for example, I will see something like this:

```
May-07 18:30:40.711 [Task monitor] DEBUG n.processor.TaskPollingMonitor - Task completed > TaskHandler[id: 1; name: sayHello (1); status: COMPLETED; exit: 0; error: -; workDir: /Users/mribeirodantas/work/3d/974e557ecaf9978f6d49966aac96ca]
```

in my `nextflow.log` file which shows the _COMPLETED_ status for one of the tasks from the `sayHello` process.

**Comment by warrenson** [comment]
Try `nextflow log -f name,tag,duration,status RUN_NAME`.

`RUN_NAME` is the name of your pipeline run, which is shown in the output (column 3) of `nextflow log`.

The format `-f` argument list the fields you want to see. (see all available output fields with `nextflow log -l`).

**Comment by aiqc** [comment]
If your tasks are containerized, you can give your docker containers a name `job_task_timeStarted` then you can use `docker ps` or Docker Desktop UI to look at their live resource utilization.
